scripts/setVersion.js

```diff
@@ -75,3 +80,7 @@ if (process.argv.length > 2) {
 // include update to the root package.json
 const packages = getFiles('package.json', '.');
 updateMgtDependencyVersion(packages, version);
+
+// include update to sonar-project.properties
+const sonarProperties = getFiles('sonar-project.properties', '.');
+updateMgtDependencyVersion(sonarProperties, version);
```

Comment:
Nit: do this in its own function `updateSonarVersion` so that it has its own logs and you can do away with the switch in the Mgt specific package updates.